ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -174,7 +178,11 @@ def _create_config(self, alignment_metadata_path: str) -> dict[str, Any]:
         volume_info = tomogram.get_output_volume_info()
         voxel_size = round(volume_info.voxel_size, 3)
         resolution = (voxel_size * 1e-10,) * 3
-        layers = [self._to_tomogram_layer(tomogram, volume_info, resolution)]
+        t = time()
+        print("Start contrast limit computation for", tomogram)
+        contrast_limits = tomogram.get_contrast_limits()
+        print("Took", (time() - t), "s")
```

Comment:
I think we could combine the print for the computed contrast limit and the time taken into one to reduce the number of prints